index.html

```diff
@@ -41,7 +42,7 @@ <h1 id="heading">I’m Joy<br>
 
    <div class="smedia" title="social-media-accounts">
    
-   <h5 id="heading5">LET’S CONNECT</h5>
+   <h5> <a id="heading5" href="#">LET’S CONNECT </a></h5>
```

Comment:
- Please, kindly remove the h5 and leave it as a link as pointed out by the previous reviewer.